src/frontend/komponenter/Fagsak/Saksoversikt/Saksoversikt.tsx

```diff
@@ -196,7 +196,7 @@ const Saksoversikt: React.FunctionComponent<IProps> = ({ fagsak }) => {
                                                 children={`${
                                                     behandling
                                                         ? behandlingsresultater[behandling.resultat]
-                                                              .navn
+                                                              .navn // TODO: Finnes resultat på dette objektet? Jeg (Charlie) klarer ikke å se at denne ligger i datamodellen hverken i backend eller frontend.
```

Comment:
Her finnes ikke `resultat` som property i IBehandling - ikke fant jeg det på serversiden heller, men tør ikke endre ettersom jeg ikke vet hva som skal gjelde her.

Reply:
Godt poeng! Her trenger vi nok en funksjon som i backend som henter det samlede resultatet (eller om backend sender med det til frontend). 

Hva tenker du er best?

Reply:
Vi legger inn `samletResultat` i `RestBehandling` på backend som gir denne verdien. Selv om det ikker er kompleks kode så virker det litt unødvendig å duplisere denne logikken på frontend for dette behovet.